test/object-store/util/test_file.hpp

```diff
@@ -108,15 +108,11 @@ void on_change_but_no_notify(realm::Realm& realm);
 #endif // TEST_ENABLE_SYNC_LOGGING_LEVEL
 
 
-struct TestLogger : realm::util::Logger::LevelThreshold, realm::util::Logger {
+struct TestLogger : realm::util::Logger {
```

Comment:
I've always kinda resented that the `TestLogger` is really just a `NullLogger` or `NoopLogger` can we rename it here?

Reply:
Will do

Reply:
Renamed it to `NullLogger` and put it in Logger.[hpp|cpp] since there was also a `MuteLogger` that did the same thing...